builder/dockerfile/internals.go

```diff
@@ -123,7 +122,7 @@ func (b *Builder) runContextCommand(args []string, allowRemote bool, allowLocalD
 				return err
 			}
 			defer os.RemoveAll(filepath.Dir(fi.Path()))
-			decompress = false
+			decompress := false
```

Comment:
`decompress` with value of `allowLocalDecompression` is not used in the code. 
actually it looks same with 
```
infos = append(infos, copyInfo{fi, false})
```
don't need to declaring decompress....